certbot/cli.py

```diff
@@ -69,6 +69,7 @@
   config_changes       Show changes made to server config during installation
   update_symlinks      Update cert symlinks based on renewal config file
   plugins              Display information about installed plugins
+  list_certs           Display information about certificates that Certbot knows about
```

Comment:
I like both of these suggestions! I'll go with `certificates`, and then "Display information about certs configured with Certbot".